app/src/main/java/net/skyscanner/backpack/demo/data/CalendarStoryType.kt

```diff
@@ -0,0 +1,159 @@
+/*
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.demo.data
+
+import net.skyscanner.backpack.calendar2.CalendarParams
+import net.skyscanner.backpack.calendar2.CalendarSelection
+import net.skyscanner.backpack.calendar2.CellInfo
+import net.skyscanner.backpack.calendar2.CellStatus
+import net.skyscanner.backpack.calendar2.CellStatusStyle
+import net.skyscanner.backpack.calendar2.extension.toIterable
+import org.threeten.bp.DayOfWeek
+import org.threeten.bp.LocalDate
+import org.threeten.bp.Month
+import org.threeten.bp.Period
+import org.threeten.bp.YearMonth
+import kotlin.math.roundToInt
+
+private val now = LocalDate.of(2019, 1, 1)
+private val range = now..(now + Period.ofYears(2))
+
+enum class CalendarStoryType {
```

Comment:
Extracted story types from Calendar2 story and created a helper method to generate the params  – this is to be used consistently in both stories.